style.css

```diff
@@ -503,6 +521,17 @@ h1 {
     align-self: self-start;
   }
 
+  .name-group {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    gap: 15px;
+  }
```

Comment:
- [ ] It will be better 😃 if you place the background image of the footer section like the Figma Design as you see here the background image of your footer is placed at the left little from the right side of the page, so kindly😊try to move the background image little to be at the right side of the footer to make nice like the Figma Design 👌💯
- [ ] Also consider decreasing the height between the hr line and the background image as you see here it is greater than the Figma Design, so please try to decrease the margin between them to make it nice like the Figma 💯😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/2409945c-fba6-4352-b45a-85256490f89e) |    ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/05e41f0a-4513-4087-b071-60017cac8abd)  |